src/common/tests/unit/all.ts

```diff
@@ -2,3 +2,4 @@ import './main';
 import '../../../dialog/tests/unit/Dialog';
 import '../../../slidepane/tests/unit/SlidePane';
 import '../../../label/tests/unit/Label';
+import '../../../textInput/tests/unit/TextInput';
```

Comment:
I think this should be `../../../textinput/tests/unit/TextInput` (lowercase directory name)